### PR TITLE
USA: Update region label properties

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1590.

Updates the `label` properties in the 50 region records of the United States.
No PIP work required.

### CODE REVIEW

File: data/856/886/45/85688645.geojson [hunk 1]
```diff
@@ -14,17 +14,15 @@
...
-        "Massachusetts State"
+        "Massachusetts"
     ],
     "label:eng_x_preferred_placetype":[
         "state"
     ],
     "label:eng_x_variant_longname":[
-        "State of Massachusetts"
+        "State of Massachusetts",
+        "Massachusetts State"
```
Comment (nvkelso) [inline data/856/886/45/85688645.geojson]: This is technically a commonwealth, so add that as variants, too, like:

```
        "Commonwealth of Massachusetts",
        "Massachusetts Commonwealth",
        "State of Massachusetts",
        "Massachusetts State"
```

File: data/856/884/81/85688481.geojson [hunk 2]
```diff
@@ -14,17 +14,15 @@
...
-        "Pennsylvania State"
+        "Pennsylvania"
     ],
     "label:eng_x_preferred_placetype":[
         "state"
     ],
     "label:eng_x_variant_longname":[
-        "State of Pennsylvania"
+        "State of Pennsylvania",
+        "Pennsylvania State"
```
Comment (nvkelso) [inline data/856/884/81/85688481.geojson]: nit: please also add the commonwealth variants for PA

File: data/856/886/41/85688641.geojson [hunk 3]
```diff
@@ -14,17 +14,15 @@
...
-        "Kentucky State"
+        "Kentucky"
     ],
     "label:eng_x_preferred_placetype":[
         "state"
     ],
     "label:eng_x_variant_longname":[
-        "State of Kentucky"
+        "State of Kentucky",
+        "Kentucky State"
```
Comment (nvkelso) [inline data/856/886/41/85688641.geojson]: nit: please also include the commonwealth variants

File: data/856/887/47/85688747.geojson [hunk 4]
```diff
@@ -14,17 +14,15 @@
...
-        "Virginia State"
+        "Virginia"
     ],
     "label:eng_x_preferred_placetype":[
         "state"
     ],
     "label:eng_x_variant_longname":[
-        "State of Virginia"
+        "State of Virginia",
+        "Virginia State"
```
Comment (nvkelso) [inline data/856/887/47/85688747.geojson]: nit: please also add the commonwealth variants.

**Comment by stepps00** [comment]
Newest commit added commonwealth variant label values, merging.

**Pull request merged**